src/qml/Mysthea/Components/qmldir

```diff
@@ -10,3 +10,5 @@ RulebookButton 1.0 RulebookButton.qml
 RoundButton 1.0 RoundButton.qml
 TextIconComboBox 1.0 TextIconComboBox.qml
 ProjectButton 1.0 ProjectButton.qml
+AppContainer 1.0 AppContainer.qml
+Header 1.0 Header.qml
```

Comment:
Because of those are components that are shared from all game application, it's better move them inside `style/TheFall/Components/` directory.

NOTE: When you move these components you have to change the `qtquickcontrols2.conf` file too because there is an error in it. Replace `Style=TheFall` with `Style=:/style/TheFall`.